app/goal_routes.py

```diff
@@ -0,0 +1,101 @@
+from app import db
+from flask import Blueprint, make_response, jsonify, abort, request 
```

Comment:
👍🏾 These imports help a great deal when it comes to our request & response cycle